.github/workflows/build-translation-source.yml

```diff
@@ -0,0 +1,69 @@
+name: Build Translation Source
+
+on:
+  push:
+    branches: [main]
+    paths-ignore:
+      - 'resources/normal/base/lang/**/*.po'
+      - 'resources/normal/base/lang/tintin.pot'
+  pull_request:
+    branches: [main]
+    paths:
+      - '.github/workflows/build-firmware.yml'
+      - 'resources/**'
+      - '!resources/normal/base/lang/**/*.po' # Ignore changes in .po files
+      - '!resources/normal/base/lang/tintin.pot' # Ignore pot file as well
+      - 'sdk/**'
+      - 'src/**'
+      - 'stored_apps/**'
+      - 'tools/**'
+      - 'third_party/**'
+      - 'waftools/**'
+      - 'waf'
+      - 'wscript'
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+
+    container:
+      image: ghcr.io/pebble-dev/pebbleos-docker:v1
+
+    strategy:
+      matrix:
+        board: ["asterix"] # POT file is universal, so only need to build for one board
```

Comment:
you don't really need a matrix build in that case, just define this as a variable above maybe